### ISSUE TITLE
A way to sleep or set spawnpoint in the pocket dimension

### ISSUE THREAD

**Issue by LukeAether16** [soulbound-devs/hammerspace#1]
Doing this also means that should have a method of leaving the dimension without a controller, not dropping the item on death, or transfer data between controllers.
Your mod have a Lot of potential, thanks.

**Comment by vaylor27** [comment]
Yeah that's a great idea! What do you think, maybe a `/return` command that does the same as using the controller used to enter the dimension?

**Comment by vaylor27** [comment]
Thank you for the compliment! You made my day. I never thought someone would actually care about my mod!
